scripts/uniqify-module-names.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import shutil
+import os
+import sh
+
+
+
```

Comment:
Nit: Consistent spacing

Side note: Maybe having any new files committed to the repo use Python Black formatting makes sense.